src/helpers/common.ts

```diff
@@ -0,0 +1,13 @@
+export const capitalizeFirstLetter = (string: string): string =>
+  string[0].toUpperCase() + string.slice(1);
+
+export const cutOffFirstLetter = (string: string) => string.slice(1);
+
+export const getErrors = (errors: string[]) =>
+  errors.reduce((acc: Record<string, string>, current?: string) => {
```

Comment:
I would use more meaningful names for `acc, current`

Reply:
fixed